Portal/src/Datahub.Portal/Pages/Project/DataProject/ProjectTools/ProjectToolListing.razor

```diff
@@ -148,7 +148,11 @@
 
         var allResources = await ProjectResourcesListingService.GetResourcesForProject(ProjectAcronym);
         _activeResources = allResources.SelectMany(a => a.GetActiveResources()).ToList();
-        _inactiveResources = allResources.Select(a => a.GetInactiveResource()).Where(a => a != null).Select(a => a.Value).ToList();
+        _inactiveResources = allResources
+            .Select(a => a.GetInactiveResource())
```

Comment:
There is something odd here, GetInactiveResources() return nulls?

Reply:
Yeah, I just did some formatting changes to get a better idea of what it looked like. I think it has to do with the backwards compatibility that existed when moving from config definitions and how it would throw a null if it wasn't found?

Not sure but I didn't want to change anything 😒